present/quote.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package present
+
+import (
+	"fmt"
+	"strings"
+)
+
+func init() {
+	Register("quote", parseQuote)
+}
+
+type Quote struct {
+	Text     string
+	Citation string
+}
+
+func (c Quote) TemplateName() string { return "quote" }
```

Comment:
warning: exported method Quote.TemplateName should have comment or be unexported (golint)